docs/website/views/_includes/landing/promo-banner.njk

```diff
@@ -1,15 +1,18 @@
-<div class="promo-banner mb-50">
-  <div class="promo-banner-container container">
-    <div class="promo-banner-block">
-      <h2>{{ landing.promo.title }}</h2>
-      <p>{{ landing.promo.description }}</p>
-      <div class="promo-banner-btns">
-        <a href="{{ landing.urls.installationUrl | url }}" class="btn btn-primary">Start now</a>
-        <a href="{{ landing.urls.docsUrl }}" class="btn btn-secondary" target="_blank">Read the docs</a>
-      </div>
-    </div>
-  </div>
-  <div class="promo-banner-waves">
-    {% include "static/assets/landing/waves-1.svg" %}
+{% from 'landing/banner.njk' import banner %}
+
+{% set buttons = [{
+  link : landing.urls.installationUrl,
+  text : 'Start now'    
+},
+{
+  link : landing.urls.docsUrl,
+  text : 'Read the docs'
+}] %}
+
+<div class="banner banner-promo">
+  {{ banner(landing.promo.title,landing.promo.description,buttons) }}
+  
+  <div class="banner-waves">
+    {% include "static/assets/landing/promo-banner-waves.svg" %}
   </div>
 </div>
```

Comment:
Do we need `includes` for a banner at all?
Mb it's better to move the whole component to the macros ?